selfdrive/assets/assets.qrc

```diff
@@ -14,5 +14,16 @@
   <file>offroad/icon_wifi_strength_medium.svg</file>
   <file>offroad/icon_wifi_strength_high.svg</file>
   <file>offroad/icon_wifi_strength_full.svg</file>
+
+  <file alias="languages.json">../ui/translations/languages.json</file>
+  <file alias="main_de">../ui/translations/main_de.qm</file>
+  <file alias="main_fr">../ui/translations/main_fr.qm</file>
+  <file alias="main_pt-BR">../ui/translations/main_pt-BR.qm</file>
+  <file alias="main_tr">../ui/translations/main_tr.qm</file>
+  <file alias="main_th">../ui/translations/main_th.qm</file>
+  <file alias="main_zh-CHS">../ui/translations/main_zh-CHS.qm</file>
+  <file alias="main_zh-CHT">../ui/translations/main_zh-CHT.qm</file>
+  <file alias="main_ko">../ui/translations/main_ko.qm</file>
+  <file alias="main_ja">../ui/translations/main_ja.qm</file>
```

Comment:
any way to keep the list in one spot?

Reply:
There should be a way, but that might complicate things.
adding new language is not a frequent operation. probably not a few times a year. I recommend keeping it as it is. we can Improve it later if necessary.what do you think?

Reply:
done, not too complicated